src/ofxstatement/plugins/alfabank.py

```diff
@@ -0,0 +1,128 @@
+#    AlfaBank (https://www.alfabank.ru) plugin for ofxstatement
+#
+#    Copyright 2017 Dmitry Pavlov <zeldigas@gmail.com>
+#
```

Comment:
I just see, that this code is based on one I wrote. =) Technically this is derived work and you should mention original author. (Not critical for me, but I believe there is some kind of good behavior pattern open source developer should follow.)